docs/WALKTHROUGH.md

```diff
@@ -0,0 +1,394 @@
+# Service Catalog Demonstration Walkthrough
+
+This document outlines the basic features of the service catalog by walking
+through a basic use case.
+
+## Step 0 - Prerequisites
+
+### Starting Kubernetes with DNS
+
+When using hack/local-up-cluster.sh, ensure to set KUBE_ENABLE_CLUSTER_DNS=true:
+
+```console
+KUBE_ENABLE_CLUSTER_DNS=true hack/local-up-cluster.sh -O
+```
+
+### Getting Helm and installing Tiller
+
+If you already have the helm binary, executing `helm init` should be all that's
+required. Detailed helm installation instructions are available
+[here](https://github.com/kubernetes/helm/blob/master/docs/install.md).
+
+## Step 1 - Installing the Service Catalog System
+
+The service catalog is conveniently packaged as a [helm](http://helm.sh/) 
+chart for installation.
+
+The chart is located in the [charts/catalog](../charts/catalog) directory in this
+repository, and supports a wide variety of customizations which are laid out in
+the README.md in that directory. To install the service-catalog with sensible
+defaults, execute this command from within the Kubernetes directory:
+
+```console
+helm install <path to service-catalog>/charts/catalog --name catalog --namespace catalog
+```
+
+Note: in the event you need to redeploy service catalog, perhaps to deploy
+different images a `helm delete --purge catalog` must be performed.
```

Comment:
this isn't strictly true because you can use `helm update`. for now, how about we recommend to delete first, then re-install? we can add details about `helm update` in follow-ups